src/utilities/geometry/Intersection.cpp

```diff
@@ -746,6 +827,7 @@ boost::optional<IntersectionResult> intersect(const std::vector<Point3d>& polygo
   std::vector<std::vector<Point3d>> newPolygons1;
   std::vector<std::vector<Point3d>> newPolygons2;
 
+  bool extraLogging = false;
```

Comment:
I went with constexpr bool in 2cca961e2235296f840dc89c1664a97571ea77eb. You can revisit if you think it's good to pass an optional parameter (but penalize everyone)

Reply:
The intention was to provide for extra logging without having to change the code - just change the value in the debugger. Logging gets clogged up with lots of unnecessary content otherwise.

But - whatever you prefer, definitely don't want to add an extra argument to the function

Reply:
Just for context, the reason I am wary of doing extra work all the time only for debugging purposes is that I know (from profiling and benchmarking) that the geometry stuff can take a long time and lots of resources for specific models.